lib/pubsub/topic.js

```diff
@@ -161,11 +170,11 @@ Topic.prototype.publish = function(messages, callback) {
  */
 Topic.prototype.delete = function(callback) {
   callback = callback || util.noop;
-  this.makeReq_('DELETE', 'topics/' + this.name, null, true, callback);
+  this.makeReq_('DELETE', this.name, null, null, callback);
 };
 
 /**
- * Get a list of the subscriptions registered to this topic. You may optionally
+ * Lists the name of the subscriptions for this topic. You may optionally
```

Comment:
`List the names of the subscriptions for this topic.` or something like it. Mostly looking at "name of the subscriptions", where instead, each subscription has a name.